mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/task/SolveMessageInconsistenciesServiceTest.java

```diff
@@ -494,9 +497,10 @@ void fixMessageInconsistenciesShouldResolveSuccessPartially(CassandraCluster cas
 
                 SoftAssertions.assertSoftly(softly -> {
                     softly.assertThat(imapUidDAO.retrieveAllMessages().collectList().block())
-                        .isEmpty();
+                        .hasSize(0);
                     softly.assertThat(messageIdDAO.retrieveAllMessages().collectList().block())
-                        .containsExactly(MESSAGE_1);
+                        .hasSize(1)
+                        .containsAnyElementsOf(ImmutableList.of(MESSAGE_1, MESSAGE_2));
```

Comment:
Ok I got it but it was hard to understand what was going on.

Can you please reword the commit to explain what is going on?

I suggest:

```
With throttling, inconsistency checking order cannot be enforced, thus, given two inconsistencies, and one that fails, we cannot know which one is fixed.
```

Now stepping back, I had the idea of specifying a `RunningOptions.withRate(1)` as a fix, but I think we should not expect DAOs to return fixes in an ordered fashion so I agree with the fix.